feeluown/cli/install.py

```diff
@@ -80,16 +80,16 @@ def gen_for_win_linux():
     f_path = pathlib.Path.home() / '.local/share/applications' / DESKTOP_FILE
     write_file(f_path, icon_string)
 
-    en_desktop_path = pathlib.Path.home() / 'Desktop'
-    cn_desktop_path = pathlib.Path.home() / '桌面'
-    desktop_file = None
-    if en_desktop_path.exists():
-        desktop_file = en_desktop_path / DESKTOP_FILE
-    if cn_desktop_path.exists():
-        desktop_file = cn_desktop_path / DESKTOP_FILE
-    if desktop_file:
+    try:
+        desktop_path_str = os.popen('xdg-user-dir DESSKTOP').read().split()[0]
+        assert desktop_path_str
+        desktop_path = pathlib.Path(desktop_path_str)
+        assert desktop_path.exists()
+        desktop_file = desktop_path / DESKTOP_FILE
         shutil.copy(f_path, desktop_file)
-    os.system('chmod +x {}'.format(desktop_file))
+        os.system('chmod +x {}'.format(desktop_file))
```

Comment:
Can we put these two lines outside the try...except... block?
```
shutil.copy(f_path, desktop_file)
os.system('chmod +x {}'.format(desktop_file))
```

Reply:
Sure but similiar to https://github.com/feeluown/FeelUOwn/pull/473#discussion_r611125200